io.gpodder.gPodder.json

```diff
@@ -0,0 +1,77 @@
+{
+  "app-id": "io.gpodder.gPodder",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpodder",
+  "rename-icon": "gpodder",
+  "rename-desktop-file": "gpodder.desktop",
+  "finish-args": [
+    "--socket=x11", "--share=ipc",
```

Comment:
does this work in wayland?

Reply:
Possibly. I'm blind and Wayland doesn't play nicely with Orca, so I can't confirm.